ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1594,6 +1594,8 @@ tests:
     cluster_profile: gcp-qe
     env:
       COMPUTE_NODE_TYPE: t2a-standard-4
+      COMPUTE_SERVICE_ACCOUNT: jiwei-worker-sa@openshift-qe.iam.gserviceaccount.com
```

Comment:
Suggest to update `openshift-openshift-tests-private-release-4.17__multi-stable.yaml:- as: gcp-ipi-xpn-mini-perm-fips-amd-f28` together.